vector/src/main/java/im/vector/activity/VectorRoomInviteMembersActivity.java

```diff
@@ -346,16 +359,22 @@ public boolean onCreateOptionsMenu(Menu menu) {
     public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
             case android.R.id.home:
+                startActivity(new Intent(this, VectorHomeActivity.class));
                 finish();
                 return true;
             case R.id.action_invite_members:
-                // Return the list of the members ids selected to invite for the room creation
-                Intent intent = new Intent();
-                intent.putExtra(EXTRA_OUT_SELECTED_USER_IDS, userIdsToInvite);
-                intent.putExtra(EXTRA_OUT_SELECTED_PARTICIPANT_ITEMS, participantsItemToInvite);
-                setResult(RESULT_OK, intent);
-                finish();
-                return true;
+
+                if (mMode.equals(VectorRoomCreationActivity.RoomCreationModes.NEW_ROOM)) {
+                    // Return the list of the members ids selected to invite for the room creation
```

Comment:
Update the comment with: "// Return the list of the selected users. The room will be created with them"